src/Examples/Examples.csproj

```diff
@@ -4,26 +4,34 @@
     <OutputType>Exe</OutputType>
     <TestUsesLibTorch>true</TestUsesLibTorch>
     <TestCuda>true</TestCuda>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks></TargetFrameworks>
+    <TargetFrameworks Condition="'$(IgnoreNetCore)' != 'true'">net5.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT' And '$(IgnoreNetFX)' != 'true'">net472;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">net5.0</TargetFrameworks>
     <TestUsesLibTorch>true</TestUsesLibTorch>
     <UseMLCodeAnalyzer>false</UseMLCodeAnalyzer>
     <UseStyleCopAnalyzer>false</UseStyleCopAnalyzer>
     <StartupObject>TorchSharp.Examples.Program</StartupObject>
     <IsPackable>false</IsPackable>
     <PlatformTarget>x64</PlatformTarget>
     <RootNamespace>TorchSharp.Examples</RootNamespace>
+    <DefineConstants Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">$(DefineConstants);NETFX472</DefineConstants>
```

Comment:
You don't need to define an additional constant. The SDK already defines constants when targeting netfx, and I believe the one you should use is `NET472_OR_GREATER` which will be true even if this project is retargeted to net48 for example. Same comment applies to all of the projects that you are manually defining this.